store/storetest/channel_store.go

```diff
@@ -6611,3 +6612,126 @@ func testCreateInitialSidebarCategories(t *testing.T, ss store.Store) {
 		assert.Equal(t, model.SidebarCategoryDirectMessages, res.Categories[2].Type)
 	})
 }
+
+func testDeleteSidebarCategory(t *testing.T, ss store.Store, s SqlSupplier) {
+	setupInitialSidebarCategories := func(t *testing.T, ss store.Store) (string, string) {
```

Comment:
We should also make a cleanup function, it causes other tests to fail.

Reply:
good find, thanks!

Reply:
Is cleanup still necessary in the DB tests? I thought we had something in place that used a fresh DB for each test case?

Reply:
Yeah, there are several failing tests because of this. There is no test helper here as in the app layer.